model/database/spans.yaml

```diff
@@ -2,7 +2,6 @@ groups:
   - id: trace.db.common.minimal
     extends: attributes.db.client.minimal
     type: attribute_group
-    stability: experimental
```

Comment:
just to confirm, all these traces attributes are not `experimental`? are they stable?

Reply:
great question. attributes groups are weird. They are not a telemetry signal - they just group attributes. 

Attributes, metrics, spans, resources all require stability property, but it means nothing on the attribute groups.
It's not documented in the https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/semantic-convention-groups.md#semantic-convention-groups:

> Groups that have attribute_group type do not describe semantic convention and are used for auxiliary purposes.

But if you think it's confusing or a bad idea to skip stability for them in some other way, please let me know, I want to hear it :)

Reply:
My confusion was mostly because the traces got their tag removed and the span was updated from `experimental` to `development`, but I might have missed some info about how this distinction was made.
That was pretty much me trying to find out how to check what should be `stable` vs `experimental` vs `development`